FF141 Relnote: IntersectionObserver.scrollMargin supported (#40242)
mdn/content: files/en-us/mozilla/firefox/releases/141/index.md

### APIs

- The {{domxref("PointerEvent/persistentDeviceId","persistentDeviceId")}} property of the {{domxref("PointerEvent")}} interface is now supported. This gives each pointing device that interacts with the screen a unique ID that persists for the session. It provides a reliable way to identify multiple pointing devices (such as pens) interacting with the screen simultaneously. ([Firefox bug 1968400](https://bugzil.la/1968400)).

> - The {{domxref("IntersectionObserver/scrollMargin","scrollMargin")}} property of the {{domxref("IntersectionObserver")}} interface is now supported. This adds a margin to all nested {{glossary("scroll container","scroll containers")}} within the root element of the observer, which allows targets inside those elements to be observed before (or after) they are scrolled into view — instead of at the point they first become visible. ([Firefox bug 1860030](https://bugzil.la/1860030)).

Reviewer: wdyt about:
```suggestion
- The {{domxref("IntersectionObserver/scrollMargin","scrollMargin")}} property of the {{domxref("IntersectionObserver")}} interface is now supported. It adds a margin to all nested {{glossary("scroll container","scroll containers")}} within the root element of the observer, which allows targets inside those elements to be observed before (or after) they are scrolled into view—instead of only when they first become visible. ([Firefox bug 1860030](https://bugzil.la/1860030)).
```

Author: Thank you.
